pkgs/development/libraries/agda/cubical/default.nix

```diff
@@ -2,17 +2,15 @@
 
 mkDerivation rec {
   pname = "cubical";
-  version = "0.4";
+  version = "0.5pre6b1ce0";
```

Comment:
Why are you not using unstable here, but in the other package(s)?

Reply:
No particular reason, the next force-push will harmonize the version strings :-)